tests/suites/test_suites.py

```diff
@@ -48,116 +48,59 @@ def iris(iris_clean) -> t.Tuple[Dataset, Dataset, AdaBoostClassifier]:
-
-    # >>> ... pre_pca_transformer.fit_transform(
-    # ...        dataset.features_columns().sample(n_samples, random_state=self.random_state)
-    # ...    ) ...
-    #
-    # with next exception: ValueError('all features must be in [0, 3] or [-4, 0]')
-
-    renamer = {
-        'sepal length (cm)': 0,
-        'sepal width (cm)': 1,
-        'petal length (cm)': 2,
-        'petal width (cm)': 3,
-        'target': 4
-    }
-
-    train, test = t.cast(
-        t.Tuple[pd.DataFrame, pd.DataFrame],
-        train_test_split(df, test_size=0.33, random_state=42)
-    )
-
-    train, test = (
-        Dataset(
-            train.rename(columns=renamer),
-            features=list(renamer.values())[:-1],
-            label_name=4, date_name=6, index_name=5
-        ),
-        Dataset(
-            test.rename(columns=renamer),
-            features=list(renamer.values())[:-1],
-            label_name=4, date_name=6, index_name=5
-        )
-    )
-
-    model = AdaBoostClassifier(random_state=0)
-    model.fit(train.features_columns, train.label_col)
-
-    return train, test, model
-
-
-def test_classification_suite(iris: t.Tuple[Dataset, Dataset, AdaBoostClassifier]):
-    train, test, model = iris
-    suite = suites.overall_classification_suite()
-
-    # TODO: Have to change min test samples of TrustScoreComparison
-    # suite[1].min_test_samples = 50
-    # suite[21].min_test_samples = 50
-
-    arguments = (
-        dict(train_dataset=train, test_dataset=test, model=model),
-        dict(train_dataset=train, model=model),
-        dict(test_dataset=test, model=model),
-    )
-
-    for args in arguments:
-        result = suite.run(**args)
-        validate_suite_result(result, expected_results='mixed')
-
-
-def test_overall_suite_with_datasets_that_have_non_textual_columns(
-    iris_with_non_textual_columns: t.Tuple[Dataset, Dataset, AdaBoostClassifier]
-):
-    train, test, model = iris_with_non_textual_columns
-    suite = suites.overall_suite()
-
-    # # TODO: Have to change min test samples of TrustScoreComparison
-    # suite[1].min_test_samples = 50
-
-    arguments = (
-        dict(train_dataset=train, test_dataset=test, model=model),
-        dict(train_dataset=train, model=model),
-        dict(test_dataset=test, model=model),
-    )
-
-    for args in arguments:
-        result = suite.run(**args)
-        validate_suite_result(result, expected_results='mixed')
-
-
-def test_regression_suite(
-    diabetes: t.Tuple[Dataset, Dataset],
-    diabetes_model: object
-):
-    train, test = diabetes
-    suite = suites.overall_regression_suite()
-
-    arguments = (
-        dict(train_dataset=train, test_dataset=test, model=diabetes_model),
-        dict(train_dataset=train, model=diabetes_model),
-        dict(test_dataset=test, model=diabetes_model),
-    )
-
-    for args in arguments:
-        result = suite.run(**args)
-        validate_suite_result(result, expected_results='mixed')
+# @pytest.fixture()
```

Comment:
@yromanyshyn hey, I wasn't sure why you tested the performance suites separately, and the purpose of this fixture. does it is still needed?